app/views/world_locations/show.html.erb

```diff
@@ -81,7 +81,7 @@
           </section>
         <% end %>
         <% if @statistics_publications.any? %>
-          <section id="statistics-publications" class="document-block documents-<%= document_block_counter %>">
+          <section id="statistics-publications" class="document-block documents-<%= document_block_counter %> govuk-!-margin-top-3">
             <h1><%= t('world_location.headings.statistics') %></h1>
```

Comment:
Where is this page rendered? I'm not sure about this `h1` hanging around on the page. Could it be a title component?

Reply:
Ah it appears here: <https://www.gov.uk/world/falkland-islands/news> - but yes h1 needs to go!

Reply:
Meu deus that section has a lot of h1s! If you feel like cleaning that up that'd be well good of you 😁

Reply:
Tell a lie - this page appears here: <https://www.gov.uk/world/uk-and-the-commonwealth>
i'm going to remove this change this shouldn't have bene touched.